fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/boot/WebXmlConfiguration.java

```diff
@@ -63,6 +64,8 @@ public ServletRegistrationBean jersey() {
         jerseyServletRegistration.addInitParameter("com.sun.jersey.spi.container.ContainerResponseFilters",
                 ResponseCorsFilter.class.getName());
         jerseyServletRegistration.addInitParameter("com.sun.jersey.config.feature.DisableWADL", "true");
+        jerseyServletRegistration.addInitParameter("com.sun.jersey.spi.container.ContainerResponseFilters",
+                ResponseCompressFilter.class.getName());
```

Comment:
@wkk91193  Stupid question, but can't we just enable gzip on tomcat instead?

Reply:
@avikganguly01

Reply:
Compression has overhead when applied indiscriminately.  

This leaves it to the field force application developer to request the response to be compressed based on discretion. Ex :- Two GET client/{clientId} is not distinguishable from each other based on channel type in Fineract so this is a soft workaround.